src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -712,11 +778,11 @@ public void setMetadata() {
                 : TransactionViewModel.FILLED_SLOT;
     }
 
-    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot, final LinkedHashSet<Hash> analyzedHashes)
-            throws Exception {
+    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4567264)